NuGet.Config

```diff
@@ -4,6 +4,6 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="MyGet google-dotnet-public" value="https://www.myget.org/F/google-dotnet-public/api/v3/index.json" />
+    <add key="MyGet google-dotnet-beta" value="https://www.myget.org/F/google-dotnet-public/api/v3/index.json" />
```

Comment:
Is this the most suitable name of the myget feed?
Might it be better to be called "google-dotnet-1.0.x" to match the git branch?